s2s/dset/_eng2chi.py

```diff
@@ -0,0 +1,52 @@
+import os
+
+from typing import Sequence
+
+import pandas as pd
+from sklearn.metrics import accuracy_score
+from nltk.translate.bleu_score import sentence_bleu
+
+from s2s.dset._base import BaseDset
+from s2s.path import DATA_PATH
+
+try:
```

Comment:
User need to install dependency before running the code.
So you don't need to check whether a dependecies is missing or not.